packages/mesos/build

```diff
@@ -62,7 +64,6 @@ disk_resource_script="$PKG_PATH/bin/make_disk_resources.py"
 cp /pkg/extra/make_disk_resources.py "$disk_resource_script"
 chmod +x "$disk_resource_script"
 
-# Copy the json file describing logrotate module.
-module_json="$PKG_PATH/etc/mesos-slave-modules/logrotate.json"
-mkdir -p "$(dirname "$module_json")"
-cp /pkg/extra/logrotate.json "$module_json"
+# Install agent modules.
+mkdir -p "$PKG_PATH/etc/mesos-agent-modules"
```

Comment:
Also, in this repo:
gen/dcos-config.yaml:171:  - path: /etc/mesos-slave-modules/overlay_slave_modules.json
gen/dcos-config.yaml:221:      MESOS_MODULES_DIR=/opt/mesosphere/etc/mesos-slave-modules